resources/puppetlabs/lein-ezbake/template/global/Makefile.erb

```diff
@@ -37,13 +37,6 @@ install-<%= EZBake::Config[:project] %>-termini:
 <% end -%>
 <% end -%>
 
-install-rpm-sysv-init: install-rpm-preinst
```

Comment:
stupid question, why is there `install-rpm-sysv-init` and `install-rpm-systemd`, but for Debian only `install-deb-sysv-init`?

Reply:
I question if this file is even used at all. Given teh last time it was updated was in 2015 I think it isn't.